integration-tests/pdf/pom.xml

```diff
@@ -64,10 +64,37 @@
   </build>
   <profiles>
     <profile>
-      <id>native</id>
+      <!--
+      TODO: fix and re-enable native tests
+
```

Comment:
An issue would be nice so that we do not forget.

Reply:
Ok, I've created https://github.com/apache/camel-quarkus/issues/472 not to forget.